corehq/apps/accounting/filters.py

```diff
@@ -64,10 +66,7 @@ class DomainFilter(BaseAccountingSingleOptionFilter):
 
 
 def clean_options(options):
-    return sorted(set(filter(
-        lambda option: option[1] and option[1].strip(),
-        options
-    )))
+    return sorted(set([option for option in options if option[1] and option[1].strip()]))
```

Comment:
nit: list comprehension could be a generator comprehension, which is a tiny bit more concise (square brackets can be eliminated):
```py
set(x for x in xxxx if ...)
```